model/patch_lifecycle.go

```diff
@@ -875,13 +875,22 @@ func getLoadProjectOptsForPatch(p *patch.Patch, githubOauthToken string) (*Proje
 		hash = p.GithubMergeData.HeadSHA
 	}
 
+	manifestID := ""
+	baseVersion, err := VersionFindOne(BaseVersionByProjectIdAndRevision(p.Project, p.Githash))
+	if err == nil && baseVersion != nil {
+		manifestID = baseVersion.Id
+	}
+	if p.ReferenceManifestID != "" {
+		manifestID = p.ReferenceManifestID
```

Comment:
nit: if this always overwrites `manifestID`, we could skip querying the base version by checking if it's set, like:
```go
var manifestID string
if p.ReferenceManifestID != "" {
    manifestID = p.ReferenceManifestID
} else {
    baseVersion, err := VersionFindOne(BaseVersionByProjectIdAndRevision(p.Project, p.Githash))
    // Get the manifest ID from the base version
}
```

Reply:
That looks much better, switched!